models/repo_list.go

```diff
@@ -61,6 +64,25 @@ func (repos RepositoryList) loadAttributes(e Engine) error {
 	for i := range repos {
 		repos[i].Owner = users[repos[i].OwnerID]
 	}
+
+	// Load primary language.
+	stats := make(LanguageStatList, 0, len(repos))
```

Comment:
Maybe we need another column `is_default_branch` on LanguageStat.

Reply:
Stats will be available only for default branch